src/Symfony/Component/HttpFoundation/Tests/InputBagTest.php

```diff
@@ -21,11 +21,14 @@ class InputBagTest extends TestCase
 
     public function testGet()
     {
-        $bag = new InputBag(['foo' => 'bar', 'null' => null]);
+        $bag = new InputBag(['foo' => 'bar', 'null' => null, 'int' => 1, 'float' => 1.0, 'bool' => false]);
 
-        $this->assertEquals('bar', $bag->get('foo'), '->get() gets the value of a parameter');
-        $this->assertEquals('default', $bag->get('unknown', 'default'), '->get() returns second argument as default if a parameter is not defined');
+        $this->assertSame('bar', $bag->get('foo'), '->get() gets the value of a string parameter');
+        $this->assertSame('default', $bag->get('unknown', 'default'), '->get() returns second argument as default if a parameter is not defined');
         $this->assertNull($bag->get('null', 'default'), '->get() returns null if null is set');
+        $this->assertSame(1, $bag->get('int'), '->get() gets the value of an int parameter');
+        $this->assertSame(1.0, $bag->get('float'), '->get() gets the value of a float parameter');
+        $this->assertFalse($bag->get('bool'), '->get() gets the value of a bool parameter');
```

Comment:
perhaps assert eg. $default=true is never casted to string

Reply:
Sure, sorry I didn't see your comment!